lib/font_awesome_flutter.dart

```diff
@@ -4,9270 +4,7661 @@ import 'package:flutter/widgets.dart';
 import 'package:font_awesome_flutter/src/icon_data.dart';
 export 'package:font_awesome_flutter/src/fa_icon.dart';
 export 'package:font_awesome_flutter/src/icon_data.dart';
+export 'package:font_awesome_flutter/src/fa_duotone_icon.dart';
```

Comment:
I'd like to keep `FaDuotoneIcon` out of the api for people who don't need it

Reply:
Similar to my suggestion for `tool/generate_example.dart`, we could comment this out, then  add a section to the readme that it needs to be uncommented for Pro or Duotone implementations (which are currently only a Pro feature)?  This gives the best of both worlds - it's readily available, but doesn't affect the package size for Free users.  We could add it alongside the instructions for uncommenting the `pubspec.yaml`.

Reply:
This is actually required for the example app to work.  Since it's an `export`, it shouldn't affect the file size - it's already included in the code that is generated, just not presented for use (the `readme` file doesn't have any instruction on how to make `FaDuotoneIcon` usable - it just says that it has to be used.  This exposes it for those who need it without impacting the size of the package).